tasks/replication-master-master.yml

```
@@ -0,0 +1,93 @@
+---
+- name: Ensure replication user exists on every host.
+  mysql_user:
+    name: "{{ mysql_replication_user.name }}"
+    host: "{{ mysql_replication_user.host | default('%') }}"
+    password: "{{ mysql_replication_user.password }}"
+    priv: "{{ mysql_replication_user.priv | default('*.*:REPLICATION SLAVE,REPLICATION CLIENT') }}"
+    state: present
+  when:
+    - (mysql_replication_role == 'master01') or (mysql_replication_role == 'master02')
+    - mysql_replication_user.name is defined
+    - (mysql_replication_master | length) > 0
+  tags: ['skip_ansible_galaxy']
+
+- name: Check if the slave user is defined in the remote host.
+  mysql_replication:
+    mode: getmaster
+    login_user: "{{ mysql_replication_user.name }}"
+    login_password: "{{ mysql_replication_user.password }}"
+  delegate_to: "{{ mysql_replication_master }}"
+  register: master
+  until: master.Is_Master is defined
+  retries: 5
+  delay: 10
+
+- name: Check slave replication status.
+  mysql_replication:
+    mode: getslave
+  ignore_errors: true
+  register: slave
+  when:
+    - (mysql_replication_role == 'master01') or (mysql_replication_role == 'master02')
+    - (mysql_replication_master | length) > 0
+  tags: ['skip_ansible_galaxy']
+
+- name: Stop slave if error replication status is detected.
+  mysql_replication:
+    mode: stopslave
+  register: restart_slave
+  ignore_errors: true
+  when:
+    - (slave.Slave_IO_Running is defined and slave.Slave_IO_Running != 'Yes') or (slave.Slave_IO_Running is defined and slave.Slave_SQL_Running != 'Yes')
+    - (mysql_replication_role == 'master01') or (mysql_replication_role == 'master02')
+    - (mysql_replication_master | length) > 0
+  tags: ['skip_ansible_galaxy']
+
+- name: Check master replication status.
+  mysql_replication: mode=getmaster
```

This is not consistent with previous instances of `mysql_replication`.
```suggestion
  mysql_replication:
    mode: getmaster
```